lib/exception_hunter/config.rb

```diff
@@ -16,6 +16,7 @@ class Config
     cattr_accessor :errors_stale_time, default: 45.days
     # @return [Array<Hash>] configured notifiers for the application (see {ExceptionHunter::Notifiers})
     cattr_accessor :notifiers, default: []
+    cattr_accessor :values_to_hide, default: []
```

Comment:
```suggestion
    cattr_accessor :sensitive_fields, default: []
```

Seems more intuitive